vin/app/controllers/vin/notes_controller.rb

```diff
@@ -1,77 +0,0 @@
-class Vin::NotesController < ApplicationController
-respond_to :json
-
-  # GET /vin/sub/:uid/shipments/:sid/notes
-  def index
-
-    @subscriber = Subscriber.find(params[:sub_id])
-
-    @package = @subscriber.shipments.find(params[:shipment_id]).package
-
-    notes = @package.note
-
-    render_result({ notes: [notes.to_s] })
-  end
-
-  # POST /vin/sub/:uid/shipments/:sid/notes
-  def create
-
-    @subscriber = Subscriber.find(params[:sub_id])
-
-    @package = @subscriber.shipments.find(params[:shipment_id]).package
-
-    note = params[:content]
-
-    status = @package.update_attribute(:note, note)
-
-    render_result({ errors: [status ? "" : "Problem updating note"] })
```

Comment:
It's a beautiful one liner for `if...else`, show me your sources that says it is not! I usually follow this styleguide for ruby https://github.com/styleguide/ruby
I personally find the ternary operator the way to go for a trivial and simple `if...else` like this.
As a requirement for the API, if no error, an empty error message should be returned, so i thought this the most elegant way.